### PR TITLE
[move] Restructuring Move Packages

### PR DESCRIPTION
**Pull request by wrwg**
### Description

- New package `aptos-stdlib` now contains general library files from the framework (like `table` and `small_map`)
  as well as those modules we need from the nursery.
- Our code has been switched over to use the new `std::error` (instead of the older `errors` in the nursery)
- Nursery is removed
- Older test code which depends on nursery has been removed

### Test Plan

Existing tests

---
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/2105)

### CODE REVIEW

File: aptos-move/move-examples/tests/move_unit_tests.rs [hunk 1]
```diff
@@ -46,12 +46,3 @@ fn test_hello_blockchain() {
     )]);
     run_tests_for_pkg("hello_blockchain", named_address);
 }
-
-#[test]
-fn test_message_board() {
```
Comment (movekevin) [inline aptos-move/move-examples/tests/move_unit_tests.rs:51]: Why delete this? I think the MessageBoard module still exists right?

File: aptos-move/framework/aptos-framework/sources/voting.move [hunk 2]
```diff
@@ -440,7 +441,7 @@ module aptos_framework::voting {
     }
 
     #[test(aptos_framework = @aptos_framework, governance = @0x123)]
-    #[expected_failure(abort_code = 519)]
+    #[expected_failure(abort_code = 0x10002)]
```
Comment (movekevin) [inline aptos-move/framework/aptos-framework/sources/voting.move:444]: Just curious is there a better way to map these values like 0x10002 to the line of the code that fails now?

Comment (wrwg) [reply aptos-move/framework/aptos-framework/sources/voting.move:444]: Nothing changed here (yet). We do still have the same errmap functionality. What is better that one can now constructs a hex code naturally as `0xCCRRRR` (C for category, R for reason.)

**Review by movekevin — APPROVED** [review]

File: aptos-move/move-examples/tests/move_unit_tests.rs (hunk 1 above)
Comment (zekun000) [reply aptos-move/move-examples/tests/move_unit_tests.rs:51]: I thought this is an example we have in tutorial, so we probably want to keep it

**Comment by msmouse** [comment]
Does the BucketTable belong to the stdlib, or is it deliberately left in the framework?

**Pull request merged**